### PR TITLE
Highlight, and expand active app when on detail view

### PR DESCRIPTION
**Pull request by saeedhmt**
_Edited:_

When going to the detail view of a model, find the parent app group in the sidemenu, highlight and expand it e.g@

image

Nothing selected:

image

### CODE REVIEW

File: jazzmin/static/jazzmin/js/main.js [hunk 1]
```diff
@@ -56,7 +56,14 @@
         const $changeListTable = $('#changelist .results table');
         if ($changeListTable.length && !$changeListTable.hasClass('table table-striped')) {
             $changeListTable.addClass('table table-striped');
-        }
+        };
+
+        var a_active = $('a.nav-link.active');
+        var main_li_parent = a_active.closest('li.nav-item.has-treeview');
+        var ul_child = main_li_parent.children('ul');
+
```
Comment (farridav) [inline jazzmin/static/jazzmin/js/main.js:64]: please change `var`'s to `const` and use `$` prefixes for constants that contain jquery objects

**Comment by farridav** [comment]
This looks good :) one small change to make, then happy to get this in.. thanks for your contribution

File: jazzmin/static/jazzmin/js/main.js (hunk 1 above)
Comment (farridav) [reply jazzmin/static/jazzmin/js/main.js:64]: Maybe also move this code up into the `setActiveLinks` function

Comment (saeedhmt) [reply jazzmin/static/jazzmin/js/main.js]: done

**Comment by saeedhmt** [comment]
> This looks good :) one small change to make, then happy to get this in.. thanks for your contribution

**Pull request merged**